include/rdm/param_data.h

```diff
@@ -0,0 +1,585 @@
+/******************************************************************************
+ * Copyright 2019 ETC Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************
+ * This file is a part of RDM. For more information, go to:
+ * https://github.com/ETCLabs/RDM
+ ******************************************************************************/
+
+/*! \file rdm/param_data.h
+ *  \brief Structures and packing/unpacking functions for the parameter data of standard PIDs.
+ *  \author Christian Reese*/
+#ifndef _RDM_PARAM_DATA_H_
+#define _RDM_PARAM_DATA_H_
+
+#include "etcpal/bool.h"
+#include "etcpal/error.h"
+#include "etcpal/inet.h"
+#include "rdm/message.h"
+
+/*! The maximum length of a typical string in the parameter data, including one extra byte for a null terminator. */
+#define RDMPD_STRING_MAX_LENGTH 33
+
+/*! A structure representing a typical string in the parameter data. */
+typedef struct RdmPdString
+{
+  char string[RDMPD_STRING_MAX_LENGTH]; /*!< The string data. */
+} RdmPdString;
+
+/* e1.33 Table 7-1: Required E1.20 Parameter IDs for the Default Responder */
+// Get/Set IDENTIFY_DEVICE See Section 10.11.1 of [RDM]
+etcpal_error_t rdmpd_unpack_get_resp_identify_device(const RdmParamData *pd, bool *identify_state_out);
```

Comment:
General note - having the output parameter names end in `_out` feels a little Hungarian-notation-y to me, I don't think it's necessary. The fact that it's not a `const *` parameter, combined with the documentation, should be enough to let people know that it's an output parameter. In Doxygen you can indicate this with `\param[in]` and `\param[out]`

Reply:
Ok, removed the _out endings. I will also add the [in] and [out]s to the documentation once it is written.